Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/IMixedRealityInputSource.cs

```diff
@@ -1,32 +1,83 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using System.Collections;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions;
+using System.Collections;
+using System.Collections.Generic;
 
 namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
 {
     /// <summary>
-    /// Interface for an input source.
-    /// An input source can be any user defined action that generally comes from a physical controller, sensor, or device.
+    /// Interface for a Mixed Reality Toolkit input source.
+    /// An input source can be any user defined input that generally comes from a physical controller, sensor, or device.
     /// </summary>
     public interface IMixedRealityInputSource : IEqualityComparer
     {
+        /// <summary>
+        /// The ID assigned to the Input Source
+        /// </summary>
         uint SourceId { get; }
 
+        /// <summary>
+        /// The Name assigned to the Input Source
+        /// </summary>
         string SourceName { get; }
 
+        /// <summary>
+        /// Outputs the current state of the Input Source, whether it is tracked or not. As defined by the SDK / Unity.
+        /// </summary>
+        InputSourceState InputSourceState { get; }
+
+        /// <summary>
+        /// The designated hand that the Input Source is managing, as defined by the SDK / Unity.
+        /// </summary>
+        Handedness Handedness { get; }
+
+        /// <summary>
+        /// The list of Pointers attached to the Input Source
+        /// </summary>
         IMixedRealityPointer[] Pointers { get; }
 
         /// <summary>
-        /// Returns the input info that the input source can provide.
+        /// Details the list of available buttons / interactions available from the Input Source.
+        /// </summary>
+        Dictionary<InputType,InteractionDefinition> Interactions { get; }
```

Comment:
Let's replace input type key with the Input action we added or else we can only define a single input type, when we need to do multiple.

Reply:
I think we resolved this in talk last night:

* InputType is the Physical Input Type (maybe we should rename it)
* Input Action is the Logical Action that is (should) be sent to the InputSystem

The mapping directs a specific input to output the action the dev has configured.  Multiple inputs can point to the same action.